docs/admin/region-availability.md

```diff
@@ -34,7 +37,65 @@ For more information about purchasing a Fabric subscription, see [Buy a Microsof
+| Central US          |           |             |             |      |           |           |               |
+| East Asia           |           |             | X           | X    | X         | X         | X             |
+| East US             | X         | X           | X           | X    | X         | X         | X             |
+| East US2            | X         | X           | X           | X    | X         | X         | X             |
+| France Central      | X         | X           | X           | X    | X         |           | X             |
+| Germany West Central| X         |             | X           |      | X         |           | X             |
+| Italy North         |           |             |             |      |           |           |               |
+| Japan East          | X         | X           | X           | X    | X         | X         |               |
+| Korea Central       | X         |             | X           |      | X         |           |               |
+| North Central US    | X         | X           | X           | X    | X         | X         | X             |
+| North Europe        | X         | X           | X           | X    | X         | X         | X             |
+| Norway East         |           |             |             |      | X         |           | X             |
+| Poland Central      |           |             |             |      |           |           | X             |
+| South Africa North  |           |             | X           | X    | X         | X         | X             |
+| South Africa West   |           |             |             |      |           |           |               |
+| South Central US    | X         | X           | X           | X    |           | X         | X             |
+| Southeast Asia      | X         |             | X           | X    | X         | X         | X             |
+| South India         | X         | X           |             |      |           |           |               |
+| Sweden Central      |           |             | X           |      | X         |           | X             |
+| Switzerland North   |           |             | X           | X    | X         |           | X             |
+| Switzerland West    |           |             |             |      |           |           |               |
+| UAE North           | X         | X           | X           |      | X         |           | X             |
+| West Europe         | X         | X           | X           | X    | X         | X         | X             |
+| West US             | X         | X           | X           | X    | X         | X         | X             |
+| West US2            | X         | X           | X           | X    | X         | X         | X             |
+| West US3            | X         |             | X           |      | X         | X         | X             |
+
+
 ## Related content
```

Comment:
Not all of these are related content. Can we choose three?

Reply:
I thought all the links in the article went in this list. I'm happy if we only include three.